src/calculateRentalCost.js

```diff
@@ -20,7 +20,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 6) {
+    return days * 40 - 50;
```

Comment:
Добавляй скобочки для улучшения читаемости выражений
```suggestion
    return (days * 40) - 50;
```